### DISCUSSION TITLE
More connection issues

### DISCUSSION THREAD

**Discussion by Emerbos** [thatskymod/Sky-CotL-Scripts#88] [General]
I can sign in just fine and races collect work, but when autocandles run, it goes to 1% then stops. This was a problem yesterday and today.

**Comment by thatskymod** [comment]
Here to help. I'm sorry to point out that since you closed your own thread, you are less likely to get a response.

The issue with the AutoCR becoming stuck at 1% was caused by a major Sky update.

This should now be resolved in the newest [libLangit032.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libLangit032.so).
